zos_administration/host_setup/README.md

```diff
@@ -4,10 +4,14 @@ This playbook is used to locate required dependencies (Python, ZOAU) on the
 remote z/OS host and build a suitable host configuration using the discovered
 dependencies.
 
+## Ansible Collection Requirement
+
+   IBM z/OS core collection 1.0.0 or later
+
 ## Getting Started
 
 If you are unfamiliar with playbooks, you can review our
-[detailed configuration guide](../../docs/share/configuration_guide.md) or
+[detailed configuration guide](../../../docs/share/configuration_guide.md) or
```

Comment:
The link seems to be broken. I think this is going on level too high in the hierarchy. Might want to revert back how it was: `../../docs/share/configuration_guide.md`

Reply:
Good catch @asifmahmud thank you.